ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/control/JogControlTopComponent.java

```diff
@@ -56,6 +57,13 @@ public JogControlTopComponent() {
     public void componentOpened() {
         setName(LocalizingService.JogControlTitle);
         setToolTipText(LocalizingService.JogControlTooltip);
+
+        // Try to load the new jog module
+        TopComponent jogTopComponent = WindowManager.getDefault().findTopComponent("JogTopComponent");
+        if(jogTopComponent != null && !jogTopComponent.isOpened()) {
+            jogTopComponent.open();
+            close();
+        }
```

Comment:
Hopefully disabling the module in the layer.xml makes it so you don't need to do this. Did the new jog component fail to load after changing `openAtStartup` to true?

Reply:
Yes the new module wouldn't start after changing the `openAtStartup`. It required to be activated manually.